lib/vtls/openssl.c

```diff
@@ -1567,7 +1562,6 @@ static CURLcode verifyhost(struct connectdata *conn, X509 *server_cert)
              "The actual format of the data will depend on the actual string
              type itself: for example for and IA5String the data will be ASCII"
 
-             Gisle researched the OpenSSL sources:
              "I checked the 0.9.6 and 0.9.8 sources before my patch and
              it always 0-terminates an IA5String."
```

Comment:
The phrasing on this comment no longer makes sense IMO, how about something like:
> It has been however verified that in 0.9.6 and 0.9.7, IA5String are always zero-terminated.